templates/repo/issue/view_content/context_menu.tmpl

```diff
@@ -0,0 +1,22 @@
+{{if .ctx.IsSigned}}
+<div class="item action ui pointing top right custom dropdown context-dropdown">
+	<a class="context-menu">
+		<i class="fa fa-ellipsis-h"></i>
+	</a>
+	<div class="menu">
+		{{$url := (Printf "%s%s/issues/%d#%s" AppUrl .ctx.Repository.FullName .ctx.Issue.Index .item.HashTag)}}
+		{{if .diff}}
+			{{$url = (Printf "%s%s/pulls/%d/files#%s" AppUrl .ctx.Repository.FullName .ctx.Issue.Index .item.HashTag)}}
+		{{end}}
+		<div class="item context clipboard" data-clipboard-text="{{$url}}">{{.ctx.i18n.Tr "repo.issues.context.copy_link"}}</div>
+		<div class="item context quote-reply {{if .diff}}quote-reply-diff{{end}}" data-target="{{.item.ID}}">{{.ctx.i18n.Tr "repo.issues.context.quote_reply"}}</div>
+		{{if or .ctx.Permission.IsAdmin (eq .item.Poster.ID .ctx.SignedUserID)}}
```

Comment:
I think anyone with write issue/pull permissions should be able to edit the contents. Currently the permissions checked in `UpdateIssueContent()` are:
```
	if !ctx.IsSigned || (ctx.User.ID != issue.PosterID && !ctx.Repo.CanWriteIssuesOrPulls(issue.IsPull)) {
```
I'm missing something or it seems that the old template got this wrong.

Reply:
`ctx.Repo.CanWriteIssuesOrPulls(issue.IsPull)` is not checking a user's permission, it is checking a repository's permission.  
Repository has Permission embedded in it.  

Though I'm unsure how far it goes, I only followed the rabbit hole a few clicks.  
Regardless, this shouldn't affect how it worked prior to this PR in terms of permissions.

Reply:
Well, not exactly:

https://github.com/go-gitea/gitea/blob/9ccb50f162bf58faeb5099755b6fdf21d807bdd4/routers/repo/issue.go#L317-L322

Reply:
But I agree that this PR is not changing that.

Reply:
If it's okay with you I'd like to do that in another PR after this is merged.  
Should be a straightforward change, but I don't want it to become another thing to check in this PR.  
I want to resolve JS/CSS conflicts as few times as possible. :joy: